auth0/src/main/java/com/auth0/android/myaccount/MyAccountException.kt

```diff
@@ -0,0 +1,115 @@
+package com.auth0.android.myaccount
+
+import com.auth0.android.Auth0Exception
+import com.auth0.android.NetworkErrorException
+
+public class MyAccountException @JvmOverloads constructor(
+    message: String,
+    exception: Auth0Exception? = null
+) : Auth0Exception(message, exception) {
+
+    private var code: String? = null
```

Comment:
If `code` will always contain a value (even if just `UNKNOWN_ERROR`), should it be nullable?